### PR TITLE
Add error to chrome extension when url is using the chrome protocol

### LINKED ISSUE #2186: Extension Error: Cannot access a chrome:// URL
**Lighthouse Version**: 1.6.5
**Chrome Version**: 58.0.3029.96
**URL**: chrome://newtab/
**Error Message**: Cannot access a chrome:// URL
**Stack Trace**:
 ```Error: Cannot access a chrome:// URL
    at Object.chrome.debugger.attach._ [as callback] (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:9198:15)
    at safeCallbackApply (extensions::sendRequest:21:15)
    at handleResponse (extensions::sendRequest:74:7)```

### PR DESCRIPTION
**Pull request by wardpeet**
thanks @brendankenny 
Fixes #2186
image

### CODE REVIEW

File: lighthouse-extension/app/src/popup.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ const NON_BUG_ERROR_MESSAGES = {
   // @see https://chromium.googlesource.com/chromium/src/+/5d1f214db0f7996f3c17cd87093d439ce4c7f8f1/chrome/common/extensions/chrome_extensions_client.cc#232
   'The extensions gallery cannot be scripted': 'The Lighthouse extension cannot audit the ' +
       'Chrome Web Store. If necessary, use the Lighthouse CLI to do so.',
+  'Cannot access a chrome': 'The Lighthouse extension cannot audit ' +
+      'Chrome specific urls. If necessary, use the Lighthouse CLI to do so.',
```
Comment (brendankenny) [inline lighthouse-extension/app/src/popup.js]: 'Chrome-specific urls' (needs a hyphen)

(though maybe there's a better name for these pages? @paulirish)

Comment (paulirish) [inline lighthouse-extension/app/src/popup.js]: name seems ok

Comment (wardpeet) [inline lighthouse-extension/app/src/popup.js]: Thanks for the english lesson 😆

Comment (brendankenny) [inline lighthouse-extension/app/src/popup.js]: :D

**Pull request merged**